Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:6
+EXPOSE 3000
+
+RUN apt-get update && apt-get install -y apt-utils
+RUN apt-get install -y libarchive-dev libtinyxml-dev && useradd --user-group --create-home --shell /bin/false app
```

Comment:
Can you combine these two lines and add some cleanup commands afterwards? (Have a look at [this](https://github.com/killercup/docker-ubuntu-rails/blob/803262307270ab0fe49b8ac7abae282e1450f4e6/Dockerfile#L3-L22), for an example.) This will help keep the image small.